bin/packages/build.js

```diff
@@ -64,6 +64,36 @@ function createStyleEntryTransform() {
 	} );
 }
 
+function createBlockJsonEntryTransform() {
+	const blocks = new Set;
+
+	return new Transform( {
+		objectMode: true,
+		async transform( file, encoding, callback ) {
+			const matches = /block-library\/src\/(.*)\/block.json$/.exec( file );
```

Comment:
I'd be curious if this works in Windows, where `file` may have a different slash.

See also: https://nodejs.org/api/path.html#path_path_sep

I wonder instead if we:

- ... aren't too specific about whether the file is located, just that it's `block.json` (`path.basename( file ) === 'block.json'`)
- ... consider the block name as the directory name in which the file is located (`path.dirname( file )`)

Reply:
Good catch, I've improved the regex to handle both back and forward slash as the directory separator. I think that will cover all cases.